From 3bc201da277d0075f618a5a0b455a7ff1b2e6199 Mon Sep 17 00:00:00 2001
From: Shaohui Xie <Shaohui.Xie@nxp.com>
Date: Fri, 29 Apr 2016 21:44:37 +0800
Subject: [PATCH 1330/1383] dts: powerpc: t2080rdb: fix rtc interrupt number

According to T2080RDB schematics, rtc connected to IRQ4.

Signed-off-by: Shaohui Xie <Shaohui.Xie@nxp.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/powerpc/boot/dts/fsl/t208xrdb.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/powerpc/boot/dts/fsl/t208xrdb.dtsi b/arch/powerpc/boot/dts/fsl/t208xrdb.dtsi
index 26ed4dc..fdeb992 100644
--- a/arch/powerpc/boot/dts/fsl/t208xrdb.dtsi
+++ b/arch/powerpc/boot/dts/fsl/t208xrdb.dtsi
@@ -141,7 +141,7 @@
 			rtc@68 {
 				compatible = "dallas,ds1339";
 				reg = <0x68>;
-				interrupts = <0x1 0x1 0 0>;
+				interrupts = <0x4 0x2 0 0>;
 			};
 
 			eeprom@50 {
-- 
2.8.1

